org/pr/ios-macos.ts

```diff
@@ -1,10 +1,27 @@
 import {message, warn, fail, danger} from "danger";
 
+// Example of relevant Podfile.lock portion:
+//
+// DEPENDENCIES:
+//     - Kanvas(from `https://github.com/tumblr/Kanvas-iOS.git`, branch `main`)
+//     - WordPress - Editor - iOS(~> 1.19.8)
+//     - WordPressUI(from `https://github.com/wordpress-mobile/WordPressUI-iOS`, commit `5ab5fd3dc8f50a27181cf14e101abe3599398cad`)
+function parseCommitPods(list: string[], entry: any): string[] {
```

Comment:
```suggestion
// Function used as a reducer to parse and accumulate pods which are referenced by commit.
// Uses recursion to parse transitive dependencies of each entry.
//
// @param list - The accumulated list of commit-referenced pods so far
// @param entry - an entry representing a pod listed in the lockfile.
//                Will be a `string` if the pod has no dependencies.
//                Otherwise, will be an `object`, with the pod being the single key, and the value being the dependencies
// @return The new list with any commit-referenced pod found added to the initial `list`
function parseCommitPods(list: string[], entry: any): string[] {
// Example of relevant Podfile.lock portion:
//
// DEPENDENCIES:
//     - Kanvas (from `https://github.com/tumblr/Kanvas-iOS.git`, branch `main`)
//     - WordPress-Editor-iOS (~> 1.19.8)
//     - WordPressUI (from `https://github.com/wordpress-mobile/WordPressUI-iOS`, commit `5ab5fd3dc8f50a27181cf14e101abe3599398cad`)
//     - WordPressKit (~> 6.1.0-beta):
//      - WordPressShared (from `https://github.com/test/pod2.git`, commit `e123bd0a9fef58a5897ed2101044f56a42e614c7`)
```

Reply:
I won't commit this suggestion because I think the indentation of the comment inside the function body got mangled by the `suggestion` block. It's inline with the method definition, but should be inline with the content instead.